packages/components/src/components/Alert/Alert.tsx

```diff
@@ -29,15 +23,15 @@ export type AlertProps = {
   /** Optional custom content to nest inside Alert
    * Use AlertContentColor or appropriate component props to set light/dark mode 'base' gray colors */
   children?: React.ReactNode
+  /** Optional boolean to determine whether the alert should be expandable */
+  expandable?: boolean
+  /** Optional initial state for expandable alert. Defaults to collapsed */
+  initialExpandedState?: 'collapsed' | 'expanded'
```

Comment:
1. Wonder if this should just be named something like `initializeExpanded` to just be a boolean as well? So then consumers could just drop `initializeExpanded` prop on there more simply than `initialExpandedState='expanded'` to override
2. Don't feel strongly, but perhaps we should use TS to only allow the `initial...` prop if expandable is set. Low stakes since sending it without expandable does nothing, but also slightly better for consumers to not include as a suggestion until after they've already marked it as expandable

Reply:
1. Renamed to `initializeExpanded` and changed to boolean
2. Added TS to only allow `initializeExpanded` if `expandable` is true